### PR TITLE
Blog - Building Golang with Bazel and Gazelle

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks

### CODE REVIEW

**Comment by msali123** [comment]
@balapriyac [Preview Link](https://6449643093b20535eb478b57--ephemeral-narwhal-3ac3b2.netlify.app/blog/build-golang-bazel-gazelle/)

File: blog/_posts/2023-06-05-build-golang-bazel-gazelle.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
+---
+title: "Building Golang with Bazel and Gazelle"
```
Comment (balapriyac) [inline blog/_posts/2023-06-05-build-golang-bazel-gazelle.md:2]:
```suggestion
title: "Building Golang With Bazel and Gazelle"
```

File: blog/_posts/2023-06-05-build-golang-bazel-gazelle.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+In this article, you'll learn about the basics of building Go using Bazel and Gazelle. You'll learn how to prepare a workspace, run, and test it; and how to develop a basic application using Bazel. To follow along, you must be familiar with the basics of Golang and how the Golang build process works. You also need to have the latest version of Go installed on your system.
+
+The code for this tutorial is available in this [GitHub repo](https://github.com/Shulammite-Aso/bazel-demo-app).
+
+## How Go and Bazel Work Together
+
+![Together]({{site.images}}{{page.slug}}/together.png)\
+
+Bazel can be used to build a Go project of any size, and the [`Go build`](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies) command is often sufficient for small applications. However, if your Go project is considerably large, supports code generation, or is in the same repository as code written in other languages, these variables can add multiple layers of complexity to your build process and the `Go build` command might not be sufficient.
```
Comment (balapriyac) [inline blog/_posts/2023-06-05-build-golang-bazel-gazelle.md:29]:
```suggestion
Bazel can be used to build a Go project of any size, and the [`go build`](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies) command is often sufficient for small applications. However, if your Go project is considerably large, supports code generation, or is in the same repository as code written in other languages, these variables can add multiple layers of complexity to your build process and the `go build` command might not be sufficient.
```

File: blog/_posts/2023-06-05-build-golang-bazel-gazelle.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+### Build Your src Files and Run the Application
+
+To build your `src` files, run the following Bazel command from the root of your project:
+
+~~~{.bash caption=">_"}
+bazel build //...
+~~~
+
+You should receive this error:
```
Comment (balapriyac) [inline blog/_posts/2023-06-05-build-golang-bazel-gazelle.md:156]:
```suggestion
You should get this error:
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks, Sameer!

**Comment by msali123** [comment]
@balapriyac [Changes done](https://644a23d1fb679257f02e5191--ephemeral-narwhal-3ac3b2.netlify.app/blog/build-golang-bazel-gazelle/)

**Comment by balapriyac** [comment]
@jalletto this is ready to merge.

**Pull request merged**